community/libtasn1/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr
+make
+make DESTDIR="$1" install
```

Comment:
```sh
./configure \
    --prefix=/usr

make
make DESTDIR="$1" install
```

Reply:
my bad forgot to change that from my repo. will change in a min